### PR TITLE
Add new module manageiq_policies

### PR DESCRIPTION
**Pull request by yaacov**
**SUMMARY**

ManageIQ is an open source management platform for Hybrid IT.

A clone of https://github.com/ansible/ansible/pull/31233

This change is adding:

    manageiq_policies module, responsible for policy_profiles management in ManageIQ

All the modules, including docs, tests and usage examples can be found here

Currently, the only requirement for the modules is manageiq-api-client-python

**ISSUE TYPE**

    New Module Pull Request

**COMPONENT NAME**

manageiq_policies.py (module)

**ANSIBLE VERSION**

ansible 2.5.0.0

### CODE REVIEW

**Comment by yaacov** [comment]
@zgalor @cben @simon3z @joelddiaz - please re-review / shipit.

**Comment by yaacov** [comment]
This is a clone of #31233

shipit

**Comment by cben** [comment]
shipit

**Comment by joelddiaz** [comment]
lgtm

File: lib/ansible/modules/remote_management/manageiq/manageiq_policies.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    resource_name: 'EngLab'
+    resource_type: 'provider'
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+'''
+
+RETURN = '''
```
Comment (resmo) [inline lib/ansible/modules/remote_management/manageiq/manageiq_policies.py:97]: not a blocker but I am missing some docs here, there are a few returns which gets returned under various conditions.

Comment (yaacov) [reply lib/ansible/modules/remote_management/manageiq/manageiq_policies.py:97]: 👍

Comment (yaacov) [reply lib/ansible/modules/remote_management/manageiq/manageiq_policies.py:97]: @resmo added a return example for the `list` option

**Comment by yaacov** [comment]
shipit

File: lib/ansible/modules/remote_management/manageiq/manageiq_policies.py (hunk 1 above)
Comment (resmo) [reply lib/ansible/modules/remote_management/manageiq/manageiq_policies.py:97]: hmm the key should correspond to the key you return on line https://github.com/ansible/ansible/pull/32892/files#diff-2bcb1f907ba628a71186846aee8852bdR342 IMHO 'profiles' 

IMHO or did I miss anything?
~~~yaml
profiles:
    description:
      - List current policy_profile and policies for a provider in ManageIQ
    returned: state=list
    type: list
    sample: '[
                {
                    "policies": [
                        {
                            "active": true,
                            "description": "OpenSCAP",
                            "name": "openscap policy"
                        },
                        {
                            "active": true,
                            "description": "Analyse incoming container images",
                            "name": "analyse incoming container images"
                        },
                        {
                            "active": true,
                            "description": "Schedule compliance after smart state analysis",
                            "name": "schedule compliance after smart state analysis"
                        }
                    ],
                    "profile_description": "OpenSCAP profile",
                    "profile_name": "openscap profile"
                }
            ]'
~~~

Comment (yaacov) [reply lib/ansible/modules/remote_management/manageiq/manageiq_policies.py:97]: > IMHO or did I miss anything?

It was fixed with a rebase, so you did miss it :-(, look here:
https://github.com/ansible/ansible/pull/32892/files#diff-2bcb1f907ba628a71186846aee8852bdR105

**Comment by zgalor** [comment]
shipit

File: lib/ansible/modules/remote_management/manageiq/manageiq_policies.py (hunk 1 above)
Comment (resmo) [reply lib/ansible/modules/remote_management/manageiq/manageiq_policies.py:97]: I am merging and creating a PR for you to review. :)

**Comment by resmo** [comment]
Thanks all!

**Pull request merged**